server/Cargo.lock

```diff
@@ -0,0 +1,56 @@
+[root]
```

Comment:
Should Cargo.lock be tracked?  My understanding of Cargo.lock is limited but I was under the impression that Cargo created this file and therefore anyone who pulls this code and builds with Cargo will automatically get their own copy that way?  Maybe this allows for a more efficient build if it is already defined?